server/conf/i18n/messages

```diff
@@ -563,10 +563,21 @@ validation.tooManySelections.v2=Please select at most {0}.
 label.firstName=First name
 label.lastName=Last name
 label.middleName=Middle name
+label.suffix=Suffix
```

Comment:
**suggestion (non-blocking):** How about `label.nameSuffix=Suffix`?  I think this will make it clearer about what kind of suffix it is.

Reply:
done.